guides/common/modules/ref_supported-host-operating-systems-and-architectures-in-registration.adoc

```diff
@@ -0,0 +1,25 @@
+[id="supported-host-operating-systems-and-architectures-in-registration_{context}"]
+= Supported Host Operating Systems and Architectures in Registration
```

Comment:
Is "Supported Clients" too short? I am also unsure about "In Registration" because it probably also affects "manage" and "provision".

Reply:
Well, managed can be anything that is registered or provisioned.
However, I'm not sure that the supported set of OSs is the same for provisioning and registration, I have my doubts.
The Foreman Introduction, however, mentions that those systems are known to succesfully "install", i.e. provision.
So I'm hesitating to rely on the same set of OSs to be supported by registration.

Reply:
@ekohl Can you please provide us with your view on this?

Reply:
@ekohl  And here, please ^^ :)

Reply:
> I'm hesitating to rely on the same set of OSs to be supported by registration.

And rightfully so. It is certainly more limited than what we can do with others. See https://community.theforeman.org/t/does-host-registration-support-freebsd-what-are-steps-to-register-a-freebsd-host-in-foreman/34003 for example. This doesn't mean it can't be made to work